cdap-ui/app/cdap/components/DataPrep/DataPrepSidePanel/ColumnsTab/ColumnsTab.scss

```diff
@@ -118,6 +121,49 @@ $option-hover-bg-color: #eeeeee;
         }
       }
     }
+
+    .columns-tab-toggle-all-dropdown {
+      .columns-tab-dropdown-toggle {
+        border: 0;
+        padding: 3px 0px;
+        transform: translateX(-2px);
+
+        &:hover,
+        &:focus {
+          background-color: initial;
+          outline: 0;
+        }
+
+        &.active,
+        &:active {
+          background-color: initial;
+
+          &:hover,
+          &:focus {
+            background-color: initial;
+            outline: 0;
```

Comment:
is this intentional, being nested inside `.active`?

Reply:
Seems like I don't need it with the `&:hover` and `&:focus` rules already defined on lines 131 to 135. Will push a commit to remove.